### PR TITLE
Form and DB

### PR DESCRIPTION
**Pull request by joelridden**
Basic form information for a Database and displaying in a card based on the design spec in https://wiki.canterbury.ac.nz/display/QuakeCore/Plan+for+development. Note (Just basic look for now till functionality is sorted)
image

Requires a database sqlite and a file path in the enviournment variables for running.
Includes a yaml file with each runs metadata that is not stored on dropbox.

Next Steps:
Adding functionality for download links for IM and Source data and test idea for downloading
Background process to run daily to check runs on dropbox to ensure database is up to date with the current state
Functionality for filters

### CODE REVIEW

File: cs_api/cs_api/db/db.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    """
+    Get the grid spacings from the database
+    :return: list of grid spacings
+    """
+    return sorted(
+        [grid_spacing.grid_spacing for grid_spacing in GridSpacing.query.all()]
+    )
+
+
+def get_scientific_versions():
```
Comment (claudio525) [inline cs_api/cs_api/db/db.py:37]: Whats the scientific version?

File: frontend/src/App.js [hunk 2]
```diff
@@ -1,16 +1,23 @@
...
+import { Form, Map } from "components";
 
 import "assets/App.css";
 import "bootstrap/dist/css/bootstrap.min.css";
 
 function App() {
   return (
     <div>
-      <div className="header">Simulation Data</div>
+      <div className="title">Cybershake</div>
```
Comment (claudio525) [inline frontend/src/App.js]: Why this change?

**Review by claudio525 — COMMENTED** [review]
It think `cs_api` should be renamed to `simulation_api` or `sim_api`, as from my understanding this tool is for retrieving simulation runs, and these can be historical or future. Cybershake specifically refers to future potential ruptures. Also based on our definitions its a complete set of simulations for all of NZ, whereas the data on this website are simulation runs, which are not complete.

File: cs_api/cs_api/db/db.py (hunk 1 above)
Comment (joelridden) [reply cs_api/cs_api/db/db.py:37]: Scientific version is something specific to cybershake runs, we currently have 18.6 and 20.4.1. This might need to be revisited as a filter based on any non cybershake simulation runs being used.

**Comment by joelridden** [comment]
For now have renamed the cybershake naming to simulation. Will keep in mind that additional simulation data with a different path might be added so might add some small functionality around that in later PR's for support.

**Review by claudio525 — APPROVED** [review]

**Review by sungeunbae — APPROVED** [review]
Looking good.
